contributing-device-support.md

```diff
@@ -0,0 +1,125 @@
+# Contributing new board support to resin.io
+
+Pre-requisites: yocto BSP layer for your particular board, supplying the BSP (board support package). Should be compatible with yocto daisy/fido/jethro releases.
+
+Every repository used to build a resin.io host Operating System is typically named resin-`<board-family>`. For example, consider the [resin-beaglebone][resin-beaglebone repo] repository which can be used to build a resin.io image for
+the BeagleBone Black, BeagleBone Green and BeagleBone Green WiFi boards.
```

Comment:
I think resin-intel is a bad example as most of the boards we come in contact are arm boards. So the part of the bootloader from the NUC is not the best to be given as an example, but I will do as suggested